### DISCUSSION TITLE
How to modify the layout used when the layout field is not specified?

### DISCUSSION THREAD

**Discussion by kecrily** [slidevjs/slidev#331] [Q&A]
I tried creating `default.vue` in the theme's `layouts` directory, but it still doesn't work when I don't specify `layout: default`.

According to #141, there should be nothing wrong with my way, but it's not taking effect, and I'm not sure if the method has changed in the meantime, or if this is a bug.

## Additional Information

I used the theme creation tool written in the documentation

```bash
pnpm init slidev-theme
```

This is dependencies written in `package.json`.

```json
"dependencies": {
  "@slidev/types": "^0.22.7",
  "codemirror-theme-vars": "^0.1.1",
  "prism-theme-vars": "^0.2.2",
  "theme-vitesse": "^0.1.12"
},
"devDependencies": {
  "@slidev/cli": "^0.22.7"
}
```

**Comment by antfu** [comment]
Sounds like a bug, I will convert it to issue for tracking and investigate later.
